src/scripting/run_script.cc

```diff
@@ -44,7 +44,7 @@ std::string get_file_content(FileSystem* fs, const std::string& filename) {
 
 // Runs the 'content' as a lua script identified by 'identifier' in 'L'.
 std::unique_ptr<LuaTable>
-run_string_as_script(lua_State* L, const std::string& identifier, const std::string& content) {
+run_string_as_script(lua_State* L, const std::string& identifier, const std::string& content, const bool no_pop_table = false) {
 	MutexLock m(MutexLock::ID::kLua);
 
```

Comment:
lua stack size (lua_gettop(L)) is often 0, as I expect, but sometimes 2 and even 3. Is this expected?

Reply:
It is shown as outdate, but it is still the same. In `run_string_as_script()`, after `MutexLock m(MutexLock::ID::kLua);` (or now after all locks) the stack size is not always as I expect it. Is my expectation wrong?